Maple2.File.Ingest/Mapper/AdditionalEffectMapper.cs

```diff
@@ -72,6 +80,75 @@ public class AdditionalEffectMapper : TypeMapper<AdditionalEffectMetadata> {
             Duration: modifyDuration);
     }
 
+    private static void AddEntry<KeyType, Type>(Dictionary<KeyType, Type> dictionary, KeyType key, Type value) where Type : notnull where KeyType : notnull {
```

Comment:
Let's move this to an extension helper function, seems like it's used in multiple places. I would give it a more descriptive name:


`EnumerableExtensions.cs`
```cs
public static AddIfNotDefault<K, V>(this IDictionary<K, V> dictionary, K key, V value)
```